arm-resources/locks/2016-09-01/swagger/locks.json

```diff
@@ -407,8 +596,8 @@
         "tags": [
           "ManagementLocks"
         ],
-        "operationId": "ManagementLocks_Get",
-        "description": "Gets the management lock of a scope.",
+        "operationId": "ManagementLocks_GetAtSubscriptionLevel",
```

Comment:
@vivsriaus @tianoms - 
This will be a breaking change in the SDK. It is the correct change though. Is this acceptable?